litmus-portal/frontend/src/components/InputField/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { TextField } from '@material-ui/core';
+import React from 'react';
+import useStyles from './styles';
+
+interface InputFieldProps {
+  label: string;
+  name: string;
+  type?: string;
+  value: string;
+  required: boolean;
+  formError: boolean;
+  handleChange: (event: React.ChangeEvent<{ value: string }>) => void;
+}
+
+const InputField: React.FC<InputFieldProps> = ({
+  type,
+  label,
+  name,
+  value,
+  required,
+  formError,
+  handleChange,
+}) => {
+  const classes = useStyles();
+  return (
+    <>
```

Comment:
Are these extra braces(<></>) required. As only one component is returned it might not be required.

Reply:
* [x] Fixed